worker/orca_grader/__main__.py

```diff
@@ -219,15 +218,16 @@ def handle_grading_job(grading_job: GradingJobJSON, image_name: str | None = Non
         else:
             warn_str = "The processes timed out during execution." if \
                         result.did_timeout else "An error was encountered during execution."
-            _LOGGER.warn(warn_str)
+            _LOGGER.warning(warn_str)
```

Comment:
why the rename here?  (did it work before?  should it have been `warning` all along?)

Reply:
It's a stupid deprecation thing (e.g., the `exists?` vs `exist?` thing with Ruby's `Dir`). Figured I'd get ahead of it